### PR TITLE
Update configuring-the-ip-address-using-the-virtual-machine-console.md

### LINKED ISSUE #156: GHES - Clarify that we do not support multiple network adapters
### What feature or product is affected?
GitHub Enterprise Server administration (virtual appliance network settings)

### What is the new or expected behavior?
From the Support side, it would be helpful to make it clear that we do not support multiple network adapters on GHES (we do not support adding additional network adapters to GitHub Enterprise Server). 

### How is the old or inaccurate behavior currently documented?

We could mention it in this documentation https://docs.github.com/en/enterprise-server/admin/configuration/configuring-the-ip-address-using-the-virtual-machine-console or on https://docs.github.com/en/enterprise-server/admin/configuration/configuring-network-settings.

### Who does this affect?

GHES admins

### PR DESCRIPTION
**Pull request by casals**
### Why:

Fixes #156  (partially - PR 1 of 2)

### What's being changed:

Added a Note box at the end of the page to make it clear that we do not support multiple network adapters on GHES.

### Additional information

I referenced GHES with the variable defined by data/variables/product/prodname_ghe_server . If this is not the correct product variable to use in this case, please reference the correct one as a required change to this PR.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/admin/configuration/configuring-the-ip-address-using-the-virtual-machine-console.md [hunk 1]
```diff
@@ -14,3 +14,10 @@ versions:
 4. Configure options for the protocol you chose.
   ![Menu with IP protocol options](/assets/images/enterprise/network-configuration/network-settings-selection.png)
 {% data reusables.enterprise_installation.vm-console-done %}
+
+---
```
Comment (janiceilene) [inline content/admin/configuration/configuring-the-ip-address-using-the-virtual-machine-console.md:18]: @casals Could you update this to use the callouts described in the [content style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#formatting-callouts) and move it to the top of the procedural? That way users will have all of the information before they start working through the steps. Thank you!

**Review by janiceilene — CHANGES_REQUESTED** [review]

File: content/admin/configuration/configuring-the-ip-address-using-the-virtual-machine-console.md (hunk 1 above)
Comment (casals) [reply content/admin/configuration/configuring-the-ip-address-using-the-virtual-machine-console.md:18]: Absolutely. Fair warning - I inserted quite a few notes in yesterday's PRs, so when you come across them again could you please just tag me + "callout" or something similar? I'll adjust all of them. Thanks!

Comment (casals) [reply content/admin/configuration/configuring-the-ip-address-using-the-virtual-machine-console.md]: Also - whenever I used notes at the beginning of their related context I labeled them as "Before you start", IIRC - could you please tag me on these as well? Consistency > ALL. :) Thanks!

**Review by janiceilene — APPROVED** [review]
This looks great @casals! I'll get it merged once things are green 🍏

**Pull request merged**